data/presets/presets/highway/primary_link.json

```diff
@@ -3,7 +3,7 @@
     "fields": [
         "name",
         "oneway",
-        "maxspeed",
+        "maxspeed/advisory",
```

Comment:
maxspeed=* is a valid tag on higway=primary_link in some countries and should not be removed.